client/src/pages/Search/index.tsx

```diff
@@ -1,52 +1,84 @@
+/* eslint-disable no-nested-ternary */
+import { Pagination, TextField } from '@mui/material';
+import { useState } from 'react';
 import CarCard from '../../components/CarCard';
+import CarsFilter from '../../components/CarsFilter';
+import CustomSkeleton from '../../helpers/skeleton';
+import { CarsCount, CarsRow } from '../../interfaces';
+import './style.css';
 
-const car = [{
-  id: 1,
-  image:
-  `https://images.unsplash.com/photo-1503376780353-
-7e6692767b70?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx
-8fGVufDB8fHx8&auto=format&fit=crop&w=870&q=80`,
-  carName: ' 2018 car name',
-  quality: 90,
-  price: 300,
-  mailage: 300,
-  description:
-  `Posuere consectetur est at lob ortis. Aenean eu l eo quam Pellentesque
-  ornare sem lacini ... more`,
-  isGoodPrice: false,
-
-},
-{
-  id: 2,
-  image:
-  `https://images.unsplash.com/photo-1503376780353-
-7e6692767b70?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx
-8fGVufDB8fHx8&auto=format&fit=crop&w=870&q=80`,
-  carName: ' 2018 car name',
-  quality: 90,
-  price: 300,
-  mailage: 300,
-  description:
-  `Posuere consectetur est at lob ortis. Aenean eu l eo quam Pellentesque
-   ornare sem lacini ... more`,
-  isGoodPrice: true,
-
-}];
 function Search() {
+  const [cars, setCars] = useState<CarsRow | []>([]);
+  const [pagination, setPagination] = useState <CarsCount>(1);
+  const [pageNumber, setPageIndex] = useState<number>(1);
```

Comment:
```suggestion
  const [currentPage, setCurrentPAge] = useState<number>(1);
```
use meaningful, descriptive names